code/modules/mob/living/simple_animal/abnormality/waw/sphinx.dm

```diff
@@ -0,0 +1,469 @@
+//Coded by Coxswain
+/mob/living/simple_animal/hostile/abnormality/sphinx
+	name = "Sphinx"
+	desc = "A gigantic stone feline."
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	icon_state = "sphinx"
+	icon_living = "sphinx"
+	var/icon_aggro = "sphinx_eye"
+	speak_emote = list("intones")
+	pixel_x = -16
+	base_pixel_x = -16
+	ranged = TRUE
+	maxHealth = 2000
+	health = 2000
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1.2, WHITE_DAMAGE = 0.5, BLACK_DAMAGE = 0.8, PALE_DAMAGE = 1.5)
+	stat_attack = HARD_CRIT
+	speed = 4
+	move_to_delay = 4
+	melee_damage_lower = 70
+	melee_damage_upper = 70
+	attack_sound = 'sound/abnormalities/sphinx/attack.ogg'
+	can_breach = TRUE
+	threat_level = WAW_LEVEL
+	melee_damage_type = WHITE_DAMAGE
+	armortype = WHITE_DAMAGE
+	melee_damage_lower = 100
+	start_qliphoth = 3
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(0, 0, 30, 35, 35),
+		ABNORMALITY_WORK_INSIGHT = list(10, 20, 35, 40, 40),
+		ABNORMALITY_WORK_ATTACHMENT = 0,
+		ABNORMALITY_WORK_REPRESSION = list(0, 0, 15, 20, 20),
+		"Riddle" = 0,		//These should never be used, but they're here for clarity.
+		"Make Offering" = 0
+	)
+	work_damage_amount = 12
+	work_damage_type = WHITE_DAMAGE
+
+	ego_list = list(
+		/datum/ego_datum/weapon/pharaoh,
+		/datum/ego_datum/armor/pharaoh
+		)
+	gift_type =  /datum/ego_gifts/pharaoh
+	var/can_act = TRUE
+	var/happy = FALSE
+	var/demand //variables between 1-4 for switch
+	var/work_cooldown
+	var/work_cooldown_time = 3 SECONDS
+	var/list/worked = list()
+	var/list/satisfied = list(
+				"Ipi etog sind lemanto.", "Lemantinco kom geng kaskihir etog!") //you mind big human || (Human-honor) show has (not-lazy) mind
+	var/list/angry = list(
+				"Mi cadu cef ipi sagmo!","Mi thran lemantolly quistramos!" )//I king threat you begone || I angry stupid man (body-coin)
+	var/list/translate = list(
+				"Ipi etog geng quir.", "Ipi inspuz geng quir.") //you mind paper translate || you quest paper translate
+	var/list/riddleloot = list(
+				/obj/item/golden_needle, /obj/item/canopic_jar)
+	var/curse_cooldown
+	var/curse_cooldown_time = 12 SECONDS
+	var/attack_chain
+
+//Work
+/mob/living/simple_animal/hostile/abnormality/sphinx/AttemptWork(mob/living/carbon/human/user, work_type)
+	if((work_type != "Riddle") && work_type != "Make Offering")
+		return TRUE
```

Comment:
I'm gonna say this every time, return `..()` instead of `TRUE` when doing this because returning `TRUE` makes it ignore any other checks AttemptWork runs to see if the person CAN do the work and while this may be caught elsewhere, it's still very bad practice.